assets/js/blocks/featured-product/block.js

```diff
@@ -305,11 +305,19 @@ class FeaturedProduct extends Component {
 											__html: product.name,
 										} }
 									/>
+									{ ! isEmpty( product.variation ) && (
+										<h3
+											className="wc-block-featured-product__variation"
+											dangerouslySetInnerHTML={ {
+												__html: product.variation,
+											} }
+										/>
+									) }
 									{ showDesc && (
 										<div
 											className="wc-block-featured-product__description"
 											dangerouslySetInnerHTML={ {
-												__html: product.short_description,
+												__html: product.description,
```

Comment:
I'm a bit confused with the line above and these lines:
https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/608/files#diff-a5de2396a8661313e79a4ccef54e854dR181
https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/608/files#diff-278216893e550dad32f808d060bd0b70R72
In `block.js` we are displaying the full description for both products and variations, but in the PHP code we try to display the short description when possible. Is that expected?

Reply:
Variations don't have a `short_description`, but they do have a `description`. So we prioritize short if set, otherwise fallback to the full description (which variations may have).

In our rest api (custom) we map them in the same way so `description` is either, depending on what is set:

```
'description'    => apply_filters( 'woocommerce_short_description', $product->get_short_description() ? $product->get_short_description() : wc_trim_string( $product->get_description(), 400 ) ),
```

This matches the PHP logic. It just means that block.js does not need to worry about it.

Reply:
Oh, ok. Makes sense. Thanks for the explanation @mikejolley!